### PR TITLE
change keystone misc

### PR DESCRIPTION
**Pull request by RensR**

### CODE REVIEW

File: contracts/src/v0.8/keystone/OCR3Capability.sol [hunk 1]
```diff
@@ -70,19 +53,15 @@ contract OCR3Capability is ConfirmedOwner, OCR2Abstract {
...
+    for (uint256 i = 0; i < _signers.length; ++i) {
       if (_transmitters[i] == address(0)) revert InvalidConfig("transmitter must not be empty");
       // add new signers
       bytes calldata publicKeys = _signers[i];
       uint16 offset = 0;
       uint16 len = uint16(publicKeys.length);
       // scan through public keys to validate encoded format
       while (offset < len) {
-        // solhint-disable-next-line no-unused-vars
-        uint8 keyType = uint8(publicKeys[offset]);
```
Comment (RensR) [inline contracts/src/v0.8/keystone/OCR3Capability.sol:82]: @DeividasK what was this for? Feels like it's just wasting gas?

**Review by DeividasK — APPROVED** [review]

File: contracts/src/v0.8/keystone/OCR3Capability.sol (hunk 1 above)
Comment (DeividasK) [reply contracts/src/v0.8/keystone/OCR3Capability.sol:82]: Oh, this is actually validation for multi-chain key encoding 😅

**Review by DeividasK — CHANGES_REQUESTED** [review]

File: contracts/src/v0.8/keystone/OCR3Capability.sol (hunk 1 above)
Comment (RensR) [reply contracts/src/v0.8/keystone/OCR3Capability.sol:82]: @archseer @DeividasK does this new format work? No silent reverts but explicit errors

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -90,6 +90,22 @@ core/scripts/gateway @smartcontractkit/dev-services
 /contracts/src/v0.8/vrf @smartcontractkit/dev-services
 /contracts/src/v0.8/keystone @smartcontractkit/keystone
 
+/core/gethwrappers/ccip @RensR @matYang @jhweintraub @0xsuryansh @RyanRHall
+/core/gethwrappers/functions @smartcontractkit/dev-services
+/core/gethwrappers/keystone @smartcontractkit/keystone
+/core/gethwrappers/liquiditymanager @RensR @matYang @jhweintraub @0xsuryansh @RyanRHall
+/core/gethwrappers/llo-feeds @smartcontractkit/data-streams-engineers
+/core/gethwrappers/operatorforwarder @smartcontractkit/data-feeds-engineers
+/core/gethwrappers/shared @RensR @matYang @RayXpub @elatoskinas
```
Comment (RensR) [inline .github/CODEOWNERS:99]: gethwrappers are an extension of onchain code and should be owner by the same owners as that onchain code

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -90,6 +90,22 @@ core/scripts/gateway @smartcontractkit/dev-services
...
+
+# The following don't exist yet but should. They are already included here to allow the teams to
+# set these folders up and own them immediately.
+/core/gethwrappers/keeper @smartcontractkit/dev-services
+/core/gethwrappers/upkeep @smartcontractkit/dev-services
+/core/gethwrappers/automation @smartcontractkit/dev-services
+/core/gethwrappers/l2ep @smartcontractkit/bix-ship
+/core/gethwrappers/vrf @smartcontractkit/dev-services
```
Comment (RensR) [inline .github/CODEOWNERS:107]: Some teams don't follow the correct structure yet, these teams will not be able to be codeowners of their gethwrapper logic until they do

**Review by archseer — APPROVED** [review]

**Review by jmank88 — APPROVED** [review]

**Review by DeividasK — APPROVED** [review]

**Pull request merged**